sound/soc/sof/sof-priv.h

```diff
@@ -73,6 +73,9 @@ bool sof_debug_check_flag(int mask);
 /* max number of DSP cores */
 #define SOF_MAX_DSP_NUM_CORES 8
 
+/* IPC interface version implemented in driver */
+#define SOF_ABI_VERSION SOF_ABI_VER(3, 0, 0)
```

Comment:
are we certain the kernel handles all versions back this far?

Reply:
@cujomalainey The ABI process has evolved quite a bit since ABI3.0.0, but yes, we have no known issues with backward compatibility. In early times, the MINOR was bumped many times in the a single release window and first SOF release with major set to 3 was SOF1.3 and this was ABI3.7.0. So we have no public releases that are older than this. SOF1.4  was ABI3.11 and that is probably still in use in some systems (some distros still ship old firmware binaries but users update the kernel, so this will actually happen).